<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editContactDialog</class>
 <widget class="QDialog" name="editContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit a contact</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="subjectLayout">
       <item>
        <widget class="QLabel" name="hashLabel">
         <property name="text">
          <string>Address (SHA1) of contact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hashEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="contactEditLayout">
       <item>
        <widget class="QLabel" name="nickNameLabel">
         <property name="text">
          <string>Local nickname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nickNameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="trustedCheckBox">
       <property name="text">
        <string>This contact is to be trusted in transactions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bottomButtonsBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
